### DISCUSSION TITLE
Pinia Fire?

### DISCUSSION THREAD

**Discussion by adamkpurdy** [vuejs/pinia#1273] [Ideas]
Will Pinia offer a plugin or package to match what VuexFire & VueFire achieved?

**Comment by posva** [comment]
It will happen within VueFire (https://github.com/vuejs/vuefire/tree/v3). It already works since you can bind to a ref in the next version of VueFire. It's missing the docs though

**Reply by adamkpurdy** [reply]
Thank you sir. Thanks for coming to the talk today.

**Reply by adamkpurdy** [reply]
@posva Just to be clear from your previous statement, the **VuexFire** functionality of binding directly to the store will not be offered moving forward with pinia, and only the **VueFire** approach will be available?

The **VuexFire** approach allows me to bind to the store outside of actual components which works nicely for me.

**Reply by posva** [reply]
no, it will be available, just different API, one more oriented to the composition API.
You can still bind anywhere, it's not unbound when components unmount, so that behavior shouldn't change.
